rtos/rtx2/TARGET_CORTEX_M/TARGET_RTOS_M4_M7/TOOLCHAIN_ARM/irq_cm4f.s

```diff
@@ -66,16 +66,20 @@ SVC_Context
                 CBNZ     R1,SVC_ContextSave     ; Branch if running thread is not deleted
                 TST      LR,#0x10               ; Check if extended stack frame
                 BNE      SVC_ContextSwitch
+#ifdef __FPU_PRESENT
```

Comment:
I thought we only distinguish by core, not by features, so this file will be included by all CM4 with and without fpu. To be fair the only change is that macro, so it amy be even clearer the way we do it now. https://github.com/ARM-software/CMSIS_5/pull/136